dnf-behave-tests/dnf/config.feature

```diff
@@ -299,3 +299,165 @@ Scenario: Reposdir option in dnf.conf file in host
+   When I execute dnf with args "install filesystem"
+   Then the exit code is 0
+
+
+@dnf5
+Scenario: The order of configs depends on the order of inclusions
+  Given I use repository "dnf-ci-fedora"
+    And I create file "/test/dnf1.conf" with
+        """
+        [main]
+        exclude=filesystem
+        """
+    And I create file "/test/dnf2.conf" with
+        """
+        [main]
+        exclude=,
+        """
+    And I include configuration path "/test/dnf2.conf" in dnf.conf
+    And I include configuration path "/test/dnf1.conf" in dnf.conf
+   When I execute dnf with args "install filesystem"
+   Then the exit code is 1
+    And stderr is
+        """
+        Failed to resolve the transaction:
+        Argument 'filesystem' matches only excluded packages.
+        """
+
+
+@dnf5
+Scenario: The default config overrides the included config
+  Given I use repository "dnf-ci-fedora"
+    And I configure dnf with
+        | key                | value      |
+        | exclude            | filesystem |
+    And I create file "/test/dnf.conf" with
+        """
+        [main]
+        exclude=,
+        """
+    And I include configuration path "/test/dnf.conf" in dnf.conf
+   When I execute dnf with args "install filesystem"
+   Then the exit code is 1
+    And stderr is
+        """
+        Failed to resolve the transaction:
+        Argument 'filesystem' matches only excluded packages.
+        """
+
+
+@dnf5
+Scenario: The config that includes another config overrides the included config
+  Given I use repository "dnf-ci-fedora"
+    And I create file "/test/dnf1.conf" with
+        """
+        #!include_config /test/dnf2.conf
+        [main]
+        exclude=filesystem
+        """
+    And I create file "/test/dnf2.conf" with
+        """
+        [main]
+        exclude=,
+        """
+    And I include configuration path "/test/dnf1.conf" in dnf.conf
+   When I execute dnf with args "install filesystem"
+   Then the exit code is 1
+    And stderr is
+        """
+        Failed to resolve the transaction:
+        Argument 'filesystem' matches only excluded packages.
+        """
+
+
+@dnf5
+Scenario: The including configs using globs
+  Given I use repository "dnf-ci-fedora"
+    And I create file "/test/dnf1.conf" with
+        """
+        [main]
+        exclude=filesystem
+        """
+    And I create file "/test/dnf2.conf" with
+        """
+        [main]
+        exclude=yum
+        """
+    And I create file "/test/something_else.conf" with
+        """
+        [main]
+        exclude=wget
+        """
+    And I include configuration path "/test/dnf*.conf" in dnf.conf
+# This fails, because the yum exclude overrides the filesystem exclude
+#   When I execute dnf with args "install filesystem"
+#   Then the exit code is 1
+#    And stderr is
+#        """
+#        Failed to resolve the transaction:
+#        Argument 'filesystem' matches only excluded packages.
+#        """
```

Comment:
I think the option (and probably others) should be appendable, the same way you can append it from the command line in dnf4. However, it currently isn't appendable even from the command line, so I wouldn't block the PR on it, just create an issue.